examples/echo_server/echo.mk

```diff
@@ -92,6 +90,9 @@ ${IMAGE_FILE} $(REPORT_FILE): $(IMAGES) $(SYSTEM_FILE)
 
 include ${SDDF}/util/util.mk
 include ${SDDF}/network/components/network_components.mk
+# Specify how many pbufs sDDF LWIP library requires for all clients
+SDDF_LWIP_NUM_BUFFS=512
```

Comment:
I changed the spelling. 

I agree that having this constant in the makefile not being the best option, but I am at a bit of a loss as to where to put it otherwise. 

I really don't think it should be in `lwipopts.h`, since that is an LWIP specific configuration file used by all of LWIP. It seems strange to make users put an sDDF library constant in their LWIP config file, plus the sDDF LWIP library does not actually explicitly use `lwipopts.h`, and it only depends on it implicitly via a different LWIP include.

I am also hesitant to make another config file just for this constant, since we already have config files for the other sDDF subsystems, and this one constant is the only thing required by the library. It can also be deduced from other sDDF queue information, but since each user of the library needs to define it separately, I am not sure how to incorporate it into the build system neatly...

Since we haven't finished finalising our solution as to how this library will be rebuilt for each user, I was thinking of holding off on having a definitive solution until after that is solved.